### PR TITLE
Create a home page

### PR DESCRIPTION
**Pull request by ruthie**
I'm not sure if I need another higher level component to manage this.  Currently the quiz controller is doing a lot and both very high level and very low level things, but I'm not sure how to usefully split this out state wise.

### CODE REVIEW

File: src/QuizController.js [hunk 1]
```diff
@@ -89,27 +94,43 @@ class QuizController extends Component {
...
+    }
+
+    handleStartQuizClick() {
+        this.resetQuiz()
+    }
+
     render() {
         const renderedComponent = (
-            this.state.finished
+            this.state.onStartPage
```
Comment (ruthie) [inline src/QuizController.js:114]: This nested ternary is not great but I also don't know what would be better here.

File: src/QuizController.js [hunk 2]
```diff
@@ -43,6 +45,8 @@ class QuizController extends Component {
         this.handleContinueClick = this.handleContinueClick.bind(this)
         this.handleFinishClick = this.handleFinishClick.bind(this)
         this.handleAgainClick = this.handleAgainClick.bind(this)
+        this.handleStartQuizClick = this.handleStartQuizClick.bind(this)
+        this.resetQuiz = this.resetQuiz.bind(this)
```
Comment (ruthie) [inline src/QuizController.js:50]: Is this necessary?  I'm not sure.

**Review by harrislapiroff — APPROVED** [review]
I think there's a good chance you're going to want to restructure this in light of some of the other changes you're making. This PR seems to be overloading the `QuizController` with behaviors that are only loosely related to each other.

I might see a tree structure for this app more like this:

```
App
└ HomePage
```

And then clicking homepage changes the tree to
```
App
└ QuizController
  └ Exercise components
```

Or similar. Ultimately you'll want to use something like react-router to manage this tree.

**Comment by ruthie** [comment]
closing in favor of https://github.com/ruthie/listening-app/pull/29

**Pull request closed without merging**